<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountsWindow</class>
 <widget class="QWidget" name="AccountsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QPushButton" name="save_pass">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>380</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset resource="galeries.qrc">
         <normaloff>:/image/ok.png</normaloff>:/image/ok.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="cancel_pass">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>380</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="galeries.qrc">
         <normaloff>:/image/cancel.png</normaloff>:/image/cancel.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>20</y>
         <width>431</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Thay đổi mật khẩu người dùng</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>110</y>
         <width>151</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Mật khẩu cũ:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="old_password">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>110</y>
         <width>411</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="new_password">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>210</y>
         <width>411</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>220</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Mật khẩu mới:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>300</y>
         <width>231</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Nhập lại mật khẩu mới:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="type_again">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>300</y>
         <width>411</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="vision_old">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>110</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="galeries.qrc">
         <normaloff>:/image/close_eye.png</normaloff>:/image/close_eye.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="vision_new">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>210</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="galeries.qrc">
         <normaloff>:/image/close_eye.png</normaloff>:/image/close_eye.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="vision_again">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>300</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="galeries.qrc">
         <normaloff>:/image/close_eye.png</normaloff>:/image/close_eye.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="1">
        <widget class="QLabel" name="email_tag">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="galeries.qrc">:/image/email.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Họ và tên</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="dob_tag">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="galeries.qrc">:/image/dob.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Số điện thoại</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="phone_tag">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="galeries.qrc">:/image/phone.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item row="9" column="5">
        <widget class="QPushButton" name="cancel_infor">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
         <property name="icon">
          <iconset resource="galeries.qrc">
           <normaloff>:/image/cancel.png</normaloff>:/image/cancel.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Ngày sinh</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Quê quán</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3" colspan="4">
        <widget class="QLineEdit" name="fullname">
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="9" column="6">
        <widget class="QPushButton" name="save_infor">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>OK</string>
         </property>
         <property name="icon">
          <iconset resource="galeries.qrc">
           <normaloff>:/image/ok.png</normaloff>:/image/ok.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="hometown_tag">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="galeries.qrc">:/image/hometown.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="name_tag">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="galeries.qrc">:/image/name.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="3" colspan="4">
        <widget class="QLineEdit" name="email"/>
       </item>
       <item row="8" column="4">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="3" colspan="3">
        <widget class="QDateEdit" name="dob">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="3" colspan="3">
        <widget class="QComboBox" name="hometown">
         <item>
          <property name="text">
           <string>Hà Nội</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hải Dương</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hải Phòng</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Quảng Ninh</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="7" column="3" colspan="3">
        <widget class="QLineEdit" name="phone"/>
       </item>
       <item row="0" column="3" colspan="4">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Cập nhật thông tin cá nhân</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="2">
        <widget class="QLabel" name="update_logo">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="galeries.qrc">:/image/update.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="galeries.qrc"/>
 </resources>
 <connections/>
</ui>
